resonate/stores/local.py

```diff
@@ -169,6 +177,19 @@ def loop(self) -> None:
     def step(self) -> Generator[tuple[str, Mesg], bool | None, None]:
         time = int(self._clock.time() * 1000)
 
+        # create schedules promises
+        for schedule in self.schedules.scan():
+            if time < schedule.next_run_time:
+                continue
+            promise_id = schedule.promise_id
+            if promise_id.endswith("{{.timestamp}}"):
+                promise_id = promise_id.replace("{{.timestamp}}", str(time))
+
+            durable_promise = self.promises.create(
+                id=promise_id, timeout=time + schedule.promise_timeout, ikey=None, strict=False, headers=schedule.promise_param.headers, data=schedule.promise_param.data, tags=schedule.promise_tags
+            )
+            assert durable_promise.pending
```

Comment:
I don't think we can assert this, on the server we log a warning, if we access to the logger we could do something similar?